content/cosign-3-0-available.md

```diff
@@ -0,0 +1,22 @@
++++
+title = "Cosign v3 is now available"
+author = "Zach Steindler (GitHub)"
+date = "2025-10-08"
+tags = ["sigstore", "cosign", "opensource"]
+type = "post"
+draft = false
++++
+
+The past few years have been incredible in the Sigstore ecosystem, seeing Sigstore-signed in-toto attestations be adopted by [Homebrew](https://blog.trailofbits.com/2024/05/14/a-peek-into-build-provenance-for-homebrew/) (May 2024), [PyPI](https://blog.pypi.org/posts/2024-11-14-pypi-now-supports-digital-attestations/) (November 2024), [Maven Central](https://central.sonatype.org/news/20250128_sigstore_signature_validation_via_portal/) (January 2025), model signing in [NVIDIA's NGC](https://developer.nvidia.com/blog/bringing-verifiable-trust-to-ai-models-model-signing-in-ngc/) (July 2025), and several others.
+
+These deployments make use of great Sigstore features, like the ability to verify content offline, being able to fetch new verification key material with The Update Framework, and the ability to use a [tile-based transparency log that's much easier to operate and scale](https://blog.sigstore.dev/rekor-v2-alpha/). These features were developed and deployed to client libraries (sigstore-go, sigstore-python, sigstore-java, ...) but for a variety of reasons these new capabilities weren't making it back to Cosign, the flagship CLI of the Sigstore project.
+
+A year ago, with some effort, we made it so [Cosign could verify attestations from these deployments](https://blog.sigstore.dev/cosign-verify-bundles/) with opt-in, off-by-default flags. That was a great step forward, but it only highlighted how far Cosign had fallen behind, and how we'd have to make breaking changes to support the capabilities expected by these newer deployments. We [outlined a path forward at SigstoreCon 2024](https://coffeehousecoders.org/blog/cosign_and_clients.html), and we're now ready to deliver on that plan.
+
+Cosign 2.6.x will support all the new capabilities of the recent deployments with opt-in, off-by-default flags: things like `--new-bundle-format` for having your signed material all in one place including information for offline verification, `--trusted-root` for having your verification material all in one place that supports key rotation without having to update your client software, and `--use-signing-config` so that the transparency log shards can be rotated without having to update your client. In addition, there are subcommands that can help you move to these new capabilities, like `cosign bundle create`, `cosign trusted-root create` and `cosign signing-config create`.
```

Comment:
Would this meant to say 2.5.x?

According the release notes (and my tests) `--new-bundle-format` was first supported in 2.5.0:
https://github.com/sigstore/cosign/releases/tag/v2.5.0